### PR TITLE
Output a grid when the program runs

### PR DESCRIPTION
**Pull request by orhanna14**
We want to be able to output a grid out the screen when the program runs.

This adds a passing spec for this feature.

### CODE REVIEW

File: spec/game_runner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+      GRID
+      stdout = StringIO.new
+      game_start = GameRunner.new(stdout)
+
+      game_start.run
+
+      expect(stdout.string).to eq(grid_template)
```
Comment (orhanna14) [inline spec/game_runner_spec.rb:21]: I am testing the right thing now! `stdout` is the thing that should be changed in the `run` method.

File: game_runner.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+         __ __ __
+        |  |  |  |
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+    GRID
+
+    @stdout.print(grid_template)
```
Comment (baccigalupi) [inline game_runner.rb:17]: It's a better practices to use attr_accessors vs direct use of instance variables. Why? It's more better to think of everything in OO as a message/method. Whenever you are thinking about data, you likely have a missing object. So use a reader here.

File: spec/game_runner_spec.rb (hunk 1 above)
Comment (baccigalupi) [inline spec/game_runner_spec.rb:21]: THIS IS IT!!!

File: spec/game_runner_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      grid_template = <<~GRID
+         __ __ __
+        |  |  |  |
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+      GRID
+      stdout = StringIO.new
```
Comment (orhanna14) [inline spec/game_runner_spec.rb:16]: I am using `StringIO` here so I can call the `.string` method later.

File: game_runner.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class GameRunner
+  def initialize(stdout)
+    @stdout = stdout
+  end
+
+  def run
```
Comment (baccigalupi) [inline game_runner.rb:6]: Now that you have some working code. Do another branch that is for refactors. 

Look at each of the rules and think about whether it is applicable, and if so how to meet it.

**Review by baccigalupi — APPROVED** [review]
Before moving on to the refactor PR, switch to using accessors vs instance variables.

File: game_runner.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+class GameRunner
+  def initialize(stdout)
```
Comment (orhanna14) [inline game_runner.rb:2]: I don't need to have `StringIO.new` here. When I run this file, I will be instantiating it as `GameRunner.new($stdout)`.

**Pull request merged**